src/Terminal/commands/cp.ts

```diff
@@ -50,7 +65,7 @@ export function cp(
 
       const tRes = server.writeToTextFile(dst, txtFile.text);
       if (!tRes.success) {
-        terminal.error("scp failed");
+        terminal.error("cp failed");
```

Comment:
Hmm... this is a tricky situation, I think this code is shared by both CP and SCP commands:
![image](https://user-images.githubusercontent.com/2285037/146283833-0ce22c4b-ae84-4982-890c-f83c47f5eecb.png)

So if you change it to one, it'll look wrong when someone used the other alias. Perhaps you should just change all errors to say "copy failed" and then it will make sense whether they used "cp" or "scp"?

Reply:
I believe scp is it's own command. https://github.com/danielyxie/bitburner/blob/dev/src/Terminal/commands/scp.ts

Reply:
My bad, I could have sworn they shared the same implementation under the hood!